vscode/src/qirGeneration.ts

```diff
@@ -22,9 +23,13 @@ export class QirGenerationError extends Error {
 export async function getQirForActiveWindow(): Promise<string> {
   let result = "";
   const editor = vscode.window.activeTextEditor;
-  if (!editor || !isQsharpDocument(editor.document)) {
+
+  if (
+    !editor ||
+    !(isQsharpDocument(editor.document) || isQsharpProject(editor.document.uri))
```

Comment:
The `isQsharpProject` check seems unnecessary now since we can't ever get to this function if we're not in a Q# file. 
(Since you had to remove the `qsharp.json` file type from the language registration)